usecases/data_model_usecase.go

```diff
@@ -162,42 +173,191 @@ func (usecase *DataModelUseCase) CreateDataModelField(ctx context.Context, field
 				return usecase.organizationSchemaRepository.CreateField(ctx, orgTx, table.Name, field)
 			},
 		)
-	})
-	return fieldId, err
+	}); err != nil {
+		return "", err
+	}
```

Comment:
I think for long statements like this, I prefer to have 
```go
err := ....

if err != nil {
   ...
}
```
But it's just a matter of formatting